### ISSUE TITLE
enabled access via mobile devices nextcloud app

### ISSUE THREAD

**Issue by AndyXheli** [nextcloud/officeonline#231]
Please & Thank you!!

If dev need a test server to connect ill be more then happy to give you Online office server url.

**Comment by epiphany11** [comment]
Is there a solution for editing files with oos via ios app?

THX

**Comment by andyxheli** [comment]
not yet, can you also create a ticket under the IOS github and tag this please ?

**Comment by juliushaertl** [comment]
Yes, this should be implemented using https://github.com/nextcloud/server/pull/17625 so that clients can use their existing implementation to add support.

**Comment by andyxheli** [comment]
@juliushaertl is this something that needs to be enabled on the mobile app side officeonline app side ?

**Comment by juliushaertl** [comment]
Once the more general implementation in richdocuments is done with https://github.com/nextcloud/richdocuments/pull/2405 we can adapt the office online app with that.

**Comment by juliusknorr** [comment]
@AndyXheli We can keep those open as enhancement ideas, even though we have not scheduled them.
